src/tech/v3/libs/smile/classification.clj

```diff
@@ -87,23 +88,32 @@
     :constructor #(LogisticRegression/fit ^Formula %1 ^DataFrame %2 ^Properties %3)
     :predictor double-array-predict-posterior}
 
-   ;; :decision-tree {:attributes #{:probabilities :attributes}
-   ;;                 :class-name "DecisionTree"
-   ;;                 :datatypes #{:float64-array}
-   ;;                 :name :decision-tree
-   ;;                 :options [{:name :max-nodes
-   ;;                            :type :int32
-   ;;                            :default 100}
-   ;;                           {:name :node-size
-   ;;                            :type :int32
-   ;;                            :default 1}
-   ;;                           {:name :split-rule
-   ;;                            :type :enumeration
-   ;;                            :class-type DecisionTree$SplitRule
-   ;;                            :lookup-table {:gini DecisionTree$SplitRule/GINI
-   ;;                                           :entropy DecisionTree$SplitRule/ENTROPY
-   ;;                                           :classification-error DecisionTree$SplitRule/CLASSIFICATION_ERROR}
-   ;;                            :default :gini}]}
+   :decision-tree
+   {:attributes #{:probabilities :attributes}
+    :name :decision-tree
+    :options [{:name :max-nodes
+               :type :int32
+               :default 100}
+              {:name :node-size
+               :type :int32
+               :default 1}
+              {:name :max-depth
+               :type :int32 
+               :default 20}
+              {:name :split-rule
+               ;; :type :enumeration
```

Comment:
The :type and :class-type keys are not used any where, correct ?

Reply:
correct, not any more.